.github/workflows/run-eval-pf-pipeline.yml

```diff
@@ -18,7 +23,8 @@ env:
   OPENAI_KEY: ${{ secrets.OPENAI_KEY }} 
   OPENAI_ENDPOINT: ${{ vars.OPENAI_ENDPOINT }}
   AISEARCH_KEY: ${{ secrets.AISEARCH_KEY }} 
-  AISEARCH_ENDPOINT: ${{ vars.AISEARCH_ENDPOINT }} 
+  AISEARCH_ENDPOINT: ${{ vars.AISEARCH_ENDPOINT }}
+  AISEARCH_INDEX: ${{ github.event.inputs.search_index || ${{ vars.AISEARCH_INDEX }} || 'patents' }} 
```

Comment:
It seems that `github.event.inputs.search_index` will always resolve to some value (because of the default) and the `${{ vars.AISEARCH_INDEX }}` will never take effect?

Reply:
When the pipeline is triggered automatically, the value of the input parameter will be empty.
The expression had a small syntax issue, which is resolved in one of the new commits.